src/app/actions.ts

```diff
@@ -142,3 +142,58 @@ export async function getNearbySightings({
 
   return json;
 }
+
+/*
+ * NEARBY AND RECENT SIGHTINGS ACTIONS
+ */
```

Comment:
nit: is this comment correct? I don't think this function is related to the nearby or recent sightings, it fetches articles from wikipedia 😅